lib/configSchema.js

```diff
@@ -32,7 +32,18 @@ const schema = {
       required: [],
       // User is free to add any properties for its own purpose
     },
-    plugins: { type: 'array', items: { type: 'string' } },
+    plugins: {
+      anyOf: [
+        {
+          type: 'object',
+          properties: {
+            localPath: { type: 'string' },
+            modules: { type: 'array', items: { type: 'string' } },
+          },
```

Comment:
Let's make `modules` _required_, also let's add `additionalProperties: false`, so eventual typos are picked and not ignored.

Reply:
I made these 2 changes. Thanks!